libraries/abstractions/pkcs11/test/iot_test_pkcs11.c

```diff
@@ -279,15 +281,37 @@ TEST_GROUP_RUNNER( Full_PKCS11_RSA )
 {
     #if ( pkcs11testRSA_KEY_SUPPORT == 1 )
         prvBeforeRunningTests();
+        #if ( pkcs11testIMPORT_PRIVATE_KEY_SUPPORT == 1 )
+            RUN_TEST_CASE( Full_PKCS11_RSA, AFQP_CreateObject );
+        #endif
+
+        /* Generating RSA keys is not currently supported.
```

Comment:
Would prefer to delete lines 288-292 rather than have commented-out code for something which doesn't exist yet.  If you do want to leave a reminder here, I'd suggest:

```c
/* TODO: Add support for testing RSA key generation. */
```

Reply:
Okay will do, I'll follow up in a separate PR